tee-worker/ts-tests/common/helpers.ts

```diff
@@ -56,4 +56,13 @@ export function identity(data: HexString | Uint8Array): Uint8Array {
     return u8aToU8a(data);
 }
 
-export const env_network = process.env.NODE_ENV === 'local' ? 'TestNet' : 'LitentryRococo';
+// https://github.com/paritytech/ss58-registry/blob/main/c.json
```

Comment:
I think it's better to point it to our rust codebase, as not every ss58prefix is listed in the registry JSON, e.g. `LitentryRococo`, `TestNet`.